src/System.Globalization/tests/project.lock.json

```diff
@@ -3,6 +3,17 @@
   "version": -9996,
   "targets": {
     "DNXCore,Version=v5.0": {
+      "System.AppContext/4.0.0-beta-23024": {
```

Comment:
The changes to this file aren't related to your change and looks like they were missed when the globalization tests were added, I think we should separate them out into a distinct commit/PR.